stories/data-viewer.js

```diff
@@ -0,0 +1,38 @@
+import { storiesOf } from '@storybook/vue';
+
+import { DataViewer } from '../dist/storybook/components';
+
+const stories = storiesOf('DataViewer', module);
+
+stories.add('empty', () => ({
+  components: { DataViewer },
+  template: `
+    <div style="width: 800px; height: 800px">
```

Comment:
You don't need to set up a size like this in storybook 5. There is a button in the workspace that let you resize and rotate the viewport. Therefore it's better to let the component take the space available.